src/geometry/GeoTrackView.hh

```diff
@@ -99,6 +113,7 @@ class GeoTrackView
     Real3&     pos_;
     Real3&     dir_;
     real_type& next_step_;
+    bool       valid_;
```

Comment:
Can you document the exact semantic of this state? (in the code below it does seem that the view is "usable" even when in a `valid_==false` state.

Reply:
I have renamed `valid_` to `dirty_` and inverted its logic. 

As Philippe requested, I have also introduced a bit more documentation:
```
    // Flag to trigger update of geometry information if and only if needed
    bool       dirty_;
```